test/piezo.js

```diff
@@ -785,6 +785,16 @@ exports["Piezo - I2C Backpack"] = {
     test.done();
   },
 
+  hasCustomFrequencyMethody: function(test) {
+    test.expect(1);
+    // I hope this is enough
+    var piezoFrequency = this.piezo.frequency;
+    var frequencyPrototype = Piezo.prototype.frequency;
+    test.notEqual(piezoFrequency, frequencyPrototype);
+
+    test.done();
+  },
```

Comment:
If I could suggest an _additional_ test... Make a test that does something like this: 

```js
var tone = this.sandbox.stub(this.piezo, "tone");
this.piezo.frequency(100, 100);
test.deepEqual(tone.lastCall.args, [100, 100]);
```

Which will assert that `frequency(...)` doesn't mess with the value before forwarding to `tone(...)`